trio/_core/_io_windows.py

```diff
@@ -268,6 +268,18 @@ def do_select():
                     queue.put_nowait(info)
 
     def _iocp_thread_fn(self):
+        # This thread sits calling GetQueuedCompletionStatusEx forever. To
+        # signal that it should shut down, the main thread just closes the
+        # IOCP, which causes GetQueuedCompletionStatusEx to return with an
+        # error:
+        IOCP_CLOSED_ERRORS = {
```

Comment:
Eh, could go either way, but I feel like the category being defined here is more specific to the semantics of this particular function than something that's generally meaningful.